blaze-client/src/test/scala/org/http4s/client/blaze/BlazeClientSuite.scala

```diff
@@ -96,7 +97,7 @@ class BlazeClientSuite extends BlazeClientBase {
           } yield r
         }
         .map(_.isRight)
-        .assert
+        .assertEquals(true)
```

Comment:
Why revert this? Is it because `munit-cats-effect` in this branch is older?

Reply:
Yeah there is no `assert` but `interceptMessage` works a lot better here